galician/dixital/dixital.md

```diff
@@ -5,3 +5,4 @@ A viaxe espacial a través do canal dixital comezara.
 A información recollida dos cúmulos estelares iniciaba unha nova era.
 
 [O asomarse](mortos/mortos.md)
+[Ao dar a volta](aire/aire.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.